RxJavaTemplate/app/src/main/java/co/nimblehq/rxjava/ui/screens/webview/WebViewFragment.kt

```diff
@@ -0,0 +1,46 @@
+package co.nimblehq.rxjava.ui.screens.webview
+
+import android.webkit.WebViewClient
+import androidx.fragment.app.viewModels
+import androidx.navigation.fragment.navArgs
+import co.nimblehq.rxjava.R
+import co.nimblehq.rxjava.extension.initialSetup
+import co.nimblehq.rxjava.extension.visibleOrGone
+import co.nimblehq.rxjava.lib.IsLoading
+import co.nimblehq.rxjava.ui.base.BaseFragment
+import dagger.hilt.android.AndroidEntryPoint
+import kotlinx.android.synthetic.main.fragment_webview.*
+import kotlinx.android.synthetic.main.view_loading.*
+
+@AndroidEntryPoint
+class WebViewFragment : BaseFragment() {
+
+    override val layoutRes = R.layout.fragment_webview
+
+    private val viewModel by viewModels<WebViewViewModel>()
+    private val args: WebViewFragmentArgs by navArgs()
+    private val bundle: WebViewBundle by lazy { args.bundle }
```

Comment:
Could we move `override val layoutRes = R.layout.fragment_webview` to below `private val` block?